seeds/cassandra/createTables.js

```diff
@@ -0,0 +1,37 @@
+const cassandra = require('cassandra-driver');
+const client = require('./config.js')
+
+
```

Comment:
not sure if you wanted to do this or not but cassandra is pretty great at running .cql files as scripts... you can create the entire schema with them just like .sql files :)